ffs/rpc/rpc.go

```diff
@@ -638,3 +598,44 @@ func toRPCDealErrors(des []ffs.DealError) []*DealError {
 	}
 	return ret
 }
+
+func fromRPCHotConfig(config *HotConfig) ffs.HotConfig {
```

Comment:
These two functions provide safe conversion from the nested pointer types in gRPC requests to our native Go types. Checks for `nil` objects where appropriate